samples/rbac.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cosi-controller-sa
+  # replace with non-default namespace name
+  namespace: default
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cosi-controller-manager-role
+rules:
+  - apiGroups: ["objectstorage.k8s.io"]
+    resources: ["bucketrequests", "bucketaccessrequests"]
+    verbs: ["get", "list", "watch", "create", "delete"]
```

Comment:
why is "BR/BAR create" needed for this ClusterRole?

Reply:
Fixed, I did not pay close attention and gave more permission than required. Thanks for details review on this @jeffvance